scripts/includes/common.sh

```diff
@@ -135,6 +135,9 @@ configure_st2_user () {
 
   SYSTEM_HOME=$(echo ~stanley)
 
+  if [ ! -d "${SYSTEM_HOME}/.ssh" ]; then
+  	sudo mkdir ${SYSTEM_HOME}/.ssh
```

Comment:
```suggestion
    sudo mkdir ${SYSTEM_HOME}/.ssh
```

Please use 2 spaces for indentation to be consistent with the rest of the code in file.

Reply:
@punkrokk Do you think we'll also need to set the permissions on `~/.ssh/` after dir creation if it didn't exist by default?

Reply:
Fixed